geode-core/src/upgradeTest/java/org/apache/geode/security/AuthExpirationBackwardCompatibleDUnitTest.java

```diff
@@ -366,6 +367,7 @@ public void multiUserCq() throws Exception {
     assertThat(unAuthorizedOps.get("user1")).asList().contains("DATA:READ:region:11");
   }
 
+  @Ignore("does not support this use case")
```

Comment:
If this test is not supported statically like this then it should be deleted. If the intent is to support his on versions 1.15 or newer see AssertJ `assumeThat` statement to disable based on version.